### ISSUE TITLE
"AttributeError: 'NoneType' object has no attribute 'findall'" when searching with search_artist_and_song()

### ISSUE THREAD

**Issue by rothos** [matheusfillipe/chartlyrics#1]
When running this simple code:

```
from chartlyrics import ChartLyricsClient

client = ChartLyricsClient()

x = client.search_artist_and_song("The Beatles", "You Won't See Me")
```

I get the following error (running Python 3.12 on MacBook running Sonoma):

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/homebrew/lib/python3.12/site-packages/chartlyrics/client.py", line 166, in search_artist_and_song
    return ArrayOfSearchLyricResult(response.content, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.12/site-packages/chartlyrics/client.py", line 118, in __init__
    for result in root.findall(f"{NAMESPACE}SearchLyricResult"):
                  ^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'findall'
```

I don't get this error for all search queries, only a small number (~5% maybe) of them.

I have a script that grabs lyrics of all Beatles songs, and I get the error when searching for these:
```
"All I've Got to Do"
"Because"
"Come and Get It"
"From Me to You"
"How Do You Do It?"
"I Will"
"I'll Be on My Way"
"I'll Get You"
"It's All Too Much"
"You Can't Do That"
"You Like Me Too Much"
"You Won't See Me"
```

**Comment by rothos** [comment]
Huh, the issue seems to be caused by this error from ChartsLyrics:

> SearchLyric: No valid words left in contains list, this could be caused by stop words.

(see here:)
http://api.chartlyrics.com/apiv1.asmx/SearchLyric?artist=beatles&song=because

Weird. I think the only thing to do is improve error handling in this module.

**Comment by matheusfillipe** [comment]
Thanks! chartlytics is a very shitty and lacking api, but hey it is free. I've added a custom exception and released a new version [v0.1.7](https://pypi.org/project/chartlyrics/).

**Comment by rothos** [comment]
Ha definitely. Thank you.
